buildSrc/src/main/java/com/google/firebase/gradle/plugins/ReleaseGenerator.kt

```diff
@@ -31,25 +34,80 @@ import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
 import org.gradle.kotlin.dsl.findByType
 
+@Serializable
+data class ReleaseReport(
```

Comment:
I more-so meant like this:
```kotlin
/**
 * Does stuff.
 *
 * @property changesByLibraryName contains libs which have opted into the release, and their changes
 * @property changedLibrariesWithNoChangelog contains libs not opted into the release, despite having 
 * changes
 */
@Serializable
data class ReleaseReport
```

This also helps with inline rendering for the IDE :))